.github/workflows/ci.yml

```diff
@@ -59,7 +59,7 @@ jobs:
       run: yarn build
 
     - name: Run yarn lint
-      run: yarn lint
+      run: ENDO_LINT_TYPES=SRC yarn lint
```

Comment:
I think people may be surprised for `yarn lint` in their terminal to have different results than CI.

How much slower is it? If it's tolerable I think it would be better to make `SRC` the default and have `NONE` be optional (along with `FULL`)

Reply:
Yes, I have similar thoughts but just wasn't sure how far to take things in this step.

Reply:
I've seen people get confused by CI having different results for `yarn lint` than CLI does. So I think they should stay in sync. But I also think the 50% hit is acceptable so I'd say to make `SRC` the default in this PR. Maybe run it by team-engineering for approval of the workflow impact.